added git lfs info to install

sudhanvalalit opened the pull request.

sudhanvalalit committed:
added git lfs info to install

nworbde reviewed (changes requested):
Thanks! in item 2, can you change "for the linux you are using" to "for the operating system you are using"?

sudhanvalalit committed:
Changed "linux you are using" to "operating system you are using"

Also changed "Reclone the repository if necessary." to "Reclone the dStar repository if necessary."

sudhanvalalit commented:
Done.
